dts/bindings/mipi-dsi/nxp,imx-mipi-dsi.yaml

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) 2022, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+description: NXP MCUX MIPI DSI
+
+compatible: "nxp,imx-mipi-dsi"
+
+include: mipi-dsi-host.yaml
+
+properties:
+    interrupts:
+      required: true
+
+    nxp,lcdif:
+      type: phandle
+      required: true
+      description:
+        Instance of the LCDIF peripheral.
+
+    dpi-color-coding:
+      type: string
+      enum:
+        - "16-bit config 1"
+        - "16-bit config 2"
+        - "16-bit config 3"
+        - "18-bit config 1"
+        - "18-bit config 2"
+        - "24-bit"
+      description:
+        MIPI DPI interface color coding.
+
+    dpi-pixel-packet:
+      type: string
+      enum:
+        - "16-bit"
+        - "18-bit"
+        - "18-bit loose"
+        - "24-bit"
+      description:
+        MIPI DSI pixel packet type send through DPI interface.
+
+    dpi-video-mode:
+      type: string
+      enum:
+        - "Non burst synch pulse"
+        - "Non burst synch event"
+        - "Burst"
+      description:
+        DPI video mode.
```

Comment:
video mode is one of the MIPI-DSI modes (other is command, STM32 also has a sort of a hybrid), not sure I understand this property. is this an attribute for video mode?

another comment, string enums are `usually-written-like-this`, not `Written like this`.

Reply:
Yes, this is a property for the video mode as expected by the Controller.
I have update the enums.